lib/sensu-plugin/metric/cli.rb

```diff
@@ -4,103 +4,191 @@
 module Sensu
   module Plugin
     class Metric
-      class CLI
-        class JSON < Sensu::Plugin::CLI
-          def output(obj = nil)
-            if obj.is_a?(String) || obj.is_a?(Exception)
-              puts obj.to_s
-            elsif obj.is_a?(Hash)
-              obj['timestamp'] ||= Time.now.to_i
-              puts ::JSON.generate(obj)
-            end
+      class CLI < Sensu::Plugin::CLI
+        def to_json(obj = nil)
```

Comment:
I know this was missing before your changes but whilst you have the hood open, could you also add the YARDoc comments on the `to_json` method?